game/views.py

```
@@ -2331,8 +2332,15 @@ def analyze_game_view(request):
             if not isinstance(m, str) or len(m) > MAX_MOVE_LENGTH:
                 return JsonResponse({'error': f'Move must be a string of at most {MAX_MOVE_LENGTH} characters'}, status=400)
+        
+        # Add timeout warning to response - analysis will be aborted after 10 seconds
+        ANALYSIS_TIMEOUT_SECONDS = getattr(settings, 'ANALYSIS_TIMEOUT_SECONDS', 10)
         summary = build_summary(moves, result, reason, fen_history=fen_history)
-        return JsonResponse(summary)
+        summary['analysis_timeout_seconds'] = ANALYSIS_TIMEOUT_SECONDS
+        
+        response = JsonResponse(summary)
+        response['X-Analysis-Timeout'] = str(ANALYSIS_TIMEOUT_SECONDS)
+        return response
```

Ruff (0.15.20)

[warning] Consider moving this statement to an `else` block

(TRY300)